packages/identity/src/odis/identifier-backwards-compatibility.test.ts

```diff
@@ -22,6 +22,7 @@ const authSigner: AuthSigner = {
 }
 const oldServiceContext = OdisUtilsOld.Query.getServiceContext('alfajores')
```

Comment:
test fails with when using a string for service context.
```suggestion
const oldServiceContext = OdisUtilsOld.Query.getServiceContext(OdisUtilsOld.Query.OdisContextName.ALFAJORES)
```

Reply:
hmm this change wasn't needed for me locally, looking into why that might be

Reply:
Okay, running `yarn clean` surfaced this error on my end. Nice catch!